modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,13 +4,25 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-APIs that have been protected by a Client ID enforcement policy require API consumers to create an application to hold the Client ID and Secret. This lets the app access the API as well as provide consumer analytics for an API developer.
+Requesting access registers a client and creates an API contract between an API
```

Comment:
When you request access, a client is registered and creates an API contract between an API

Reply:
Fixed.